tapchannel/aux_traffic_shaper.go

```diff
@@ -140,6 +140,33 @@ func (s *AuxTrafficShaper) PaymentBandwidth(htlcBlob,
 		return linkBandwidth, nil
 	}
 
+	// Get the minimum HTLC amount, which is just above dust.
+	minHtlcAmt := lnwire.NewMSatFromSatoshis(DefaultOnChainHtlcAmount)
```

Comment:
Hmm, looking at the previous commit, I wondered if this value needed to be dynamic, based on the channel config's dust limit.
But luckily for `lnd` to `lnd` this value is always static: https://github.com/lightningnetwork/lnd/blob/b58fcf73fff36a0ac45e89fac2d7aa589fa6c47b/lnwallet/wallet.go#L1419

Reply:
I see, perhaps worth keeping track of somehow, in case we ever have tap channel between different LN node implementations